screwdriver.yaml

```diff
@@ -2,20 +2,28 @@ workflow:
     - publish
 
 shared:
-    image: node:6
+    image: centos
 
 jobs:
     main:
         steps:
-            - install: npm install
-            - test: npm test
-
+            # To verify if the hyperctl-download script works
+            - hyperctl-download: bash -xe scripts/hyperctl-download.sh
```

Comment:
Next step is `chmod +x hyperctl`, if the file not exists, chmod will throw err saying file not exist. Is that good enough?